.github/workflows/zxc-build-library.yaml

```diff
@@ -10,94 +10,114 @@ permissions:
+        - xlarge
+        - ephemeral
+      env:
+        HEDERA_NETWORK: localhost
+      strategy:
+        matrix:
+          include:
+            - os: Linux
+              preset: linux-x64
+            # - os: macos-12
+            #   preset: macos-x64
+            # - os: windows-latest
+            #   preset: windows-x64
+      steps:
+          - name: Checkout Code
+            uses: actions/checkout@v3
+            with:
+              submodules: true
+          - name: Install Docker Compose
+            run: |
+              sudo apt-get update
+              sudo apt-get install -y \
+                  ca-certificates \
+                  curl \
+                  gnupg \
+                  lsb-release
+              sudo mkdir -p /etc/apt/keyrings
+              curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+              echo \
+                "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+                $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+              sudo apt-get update
+              sudo apt-get install -y docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+
+          - name: Check docker compose version
+            run: docker compose version
+
+          - name: Use Node.js 18
+            uses: actions/setup-node@v3
+            with:
+              node-version: 18
+        
+          - name: Install Local Node
+            run: sudo npm install -g @hashgraph/hedera-local
+      
+          - name: Start the local node
+            run: sudo npx @hashgraph/hedera-local start -d --network local
+
+          - name: Install Linux Prerequisites
+            if: ${{ runner.os == 'Linux' }}
+            run: |
+              sudo apt-get update
+              sudo apt-get install -y pkg-config libc6-dbg libgtest-dev build-essential
+        
+          - name: Ensure Binary Cache Path Exists
+            run: mkdir -p "${{ github.workspace }}/b/vcpkg_cache"
+        
+          - name: Install CMake & Ninja
+            uses: lukka/get-cmake@latest
+        
+          - name: Setup VCPkg
+            uses: lukka/run-vcpkg@v10
+            with:
+              binaryCachePath: ${{ github.workspace }}/b/vcpkg_cache
+        
+          - name: CMake Build (Debug)
+            uses: lukka/run-cmake@v10
+            env:
+              VCPKG_BINARY_SOURCES: clear
+            with:
+              configurePreset: ${{ matrix.preset }}-debug
+              buildPreset: ${{ matrix.preset }}-debug
+        
+          # - name: CMake Build (Release)
+          #   uses: lukka/run-cmake@v10
+          #   env:
+          #     VCPKG_BINARY_SOURCES: clear
+          #   with:
+          #     configurePreset: ${{ matrix.preset }}-release
+          #     buildPreset: ${{ matrix.preset }}-release
```

Comment:
We must build both release and debug of the libraries. Building release only takes a fraction of the time compared to the debug build.

Reply:
Release builds has been activated again with 29b615968f94f078ec8f9dced266e0418078085f.
I commented them only to reduce the run time during my attempts to fix the issue.
I know how important is this step and my plan was to remove the comments at the end, but I forgot.